### ISSUE TITLE
UI: Date / Datetime

### ISSUE THREAD

**Issue by benhaynes** [RNGR/Directus#862]
Add a UI option to format the listing view date (MM dd, YYYY, etc)

**Comment by WellingGuzman** [comment]
@benhaynes While trying to force Date UI to not display time, because now as an option I can enter `HH:mm:ss` on date and would display the time, wouldn't be better to merge this two into one Datetime UI? As now the format is open, time can be included or excluded.

**Comment by benhaynes** [comment]
Yes! That's much better. So there would be a single datatime UI that changes based on the datatype being: date, time, datetime, timestamp, or year? This is much better because we can really polish up that UI and add more options and it will remove a lot of duplicate code. Great thinking @WellingGuzman !

**Comment by benhaynes** [comment]
Some Options: (can anyone think of more good ones?)
* Read Only 
* Datetime Format: `HH:mm:ss`
* Show Contextual Date In List View
* Auto-Populate When Hidden And Null
* Use Dropdowns (instead of HTML5 inputs)
* Timezone?

**Comment by WellingGuzman** [comment]
This is done, but pending on merging the two UI together. @benhaynes should we open a new ticket for merging?

I'm closing it.

**Comment by benhaynes** [comment]
(nice, I opened one)
